Dshop/apps/products_catalogue/tests/test_api_products.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from django.urls import reverse
+from apps.products_catalogue.models import Category, Product
+# pylama:ignore=W0404, W0611
+from apps.users.conftest import api_client, login_url, login_data, user_instance, user_instance_token
```

Comment:
PEP8 import order rules not followed

Reply:
It's not followed in the whole project. Good pointing out, we need to add something to guard id. Right now, let's ignore.